rotkehlchen/db/utils.py

```diff
@@ -58,7 +58,7 @@ class DBAssetBalance(NamedTuple):
     amount: str
     usd_value: str
 
-    def serialize(self, export_data: Optional[Tuple[Asset, Price]] = None) -> Dict[str, str]:
```

Comment:
Why `Any`? Shouldn't it be `Union[str, int]` ? `Any` as a type is the same as disabling mypy checks.